cmd/main.go

```diff
@@ -312,7 +312,8 @@ func main() {
 	eventsHandler := createEventsHandler(crdEventsHandler, db, authzHandler, notificationStream, log)
 
 	prometheusRegistry := prometheus.DefaultRegisterer
-	metricsManager := metrics.NewMetricsManager(prometheusRegistry, eventsHandler)
+	directoryUsageMonitorConfig := metrics.NewDirectoryUsageCollectorConfig([]string{Options.WorkDir})
```

Comment:
nit: I do not see any advantages with creating a config with a new function over instantiating a public struct config instead, in my opinion the code should be as simple as possible